### DISCUSSION TITLE
Chatbox II messes up admin panel

### DISCUSSION THREAD

**Discussion by Hiroyuki0487** [e107inc/e107#5169] [Q&A Plugins]
So I installed Chatbox II because I like its functionalities, however, it messed up my admin pannel how do I fix that?

**Comment by Deltik** [comment]
Chatbox II was written for e107 v0.7 from over 15 years ago and included source code injections in the e107 core for proper functionality.  I would expect it to be quite broken by now and would not recommend attempting to use it anymore.

**Reply by Hiroyuki0487** [reply]
Do you have any good alternatives that offer the same functionality? And why does chatbox (original) work fine

**Reply by Deltik** [reply]
I'm not aware of any alternatives.  The `chatbox_menu` plugin is a core plugin that is maintained in the main e107 repository, so it is given attention by the core maintainers.

**Reply by Moc** [reply]
@Hiroyuki0487 Feel free to submit feature requests for the core chatbox plugin. There may be some already posted: 

https://github.com/e107inc/e107/issues?q=chatbox+is%3Aopen+label%3A%22plugin%3A+chatbox%22
https://github.com/e107inc/e107/issues/2196
